rootfs/bin/read_procfile_keys

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import sys
+
+keys = []
+with open(sys.argv[1], 'r') as f:
+	for line in f:
```

Comment:
I think tabs are weird in python--could you reformat with 2 spaces?

Also would it be worthwhile to refactor this into a function called in `if name == '__main__:`? And add some unit tests? 💚